examples/components/MNIST/aggregatemodelweights/run.py

```diff
@@ -82,7 +84,7 @@ def add_model(self, model_path: str):
 
         else:
             # load the new model
-            model_to_add = torch.load(model_path)
+            model_to_add = torch.load(model_path, map_location=torch.device("cpu"))
```

Comment:
This is necessary if you are training on the GPU but not running aggregation on GPU

Reply:
This forcefully loads the model to CPU instead of GPU otherwise the model is always loaded based on the device where it was trained

https://stackoverflow.com/questions/65882750/please-use-torch-load-with-map-location-torch-devicecpu